Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -134,7 +141,12 @@ public override void Awake()
 		_bumperTransform = bumper.GetComponent<Transform>();
 		_bumperRigidbody = bumper.GetComponent<Rigidbody>();
 
-		SetupStateMethodMap();
+		InitializeStateMethodMap();
+	}
+
+	void Start()
+	{
+		ChangeState( ActorStates.Grounded );
```

Comment:
The way it's setup, the expectation is that another there will be another script that acts as the controller and it will setup the state method map in `Awake()`. `ChangeState()` also sets a reference to the current method, so if it is called before the controller script has had a chance to setup the state methods then it will have a reference to the default method, which does nothing. `InitializeStateMethodMap()` just fills the method map will empty methods to prevent null references.